AprilSylph/XKit-Rewritten
# Show Originals: Fix mode in in-blog search on disabled blog

### Description


This fixes an extremely minor bug where if you're on the blog view modal of a blog you have Show Originals disabled on and do a search, the control element looks enabled (data-timeline is `/v2/blog/whatever/search`; my check was too specific.)


### Testing steps


 - add a blog to the show originals "always show reblogs from these blogs" list 
 - navigate to that blog's blog view modal/standalone blog view page
 - do a search; the show originals element should stay disabled
 - filter by a tag; the show originals element should stay disabled
